src/cf-expandables/usage.md

```diff
@@ -78,27 +78,29 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 
 ### Expanded
 
-Sometimes you may want the expandable to be open by default.
-This is as easy as adding the `.o-expandable_content__onload-open` modifier to the `.o-expandable_content` block.
+Sometimes you may want the expandable to be open by default. This is as easy
```

Comment:
L:82 was more than 80 characters though and it's all one paragraph. I don't see the purpose of restricting a line break on punctuation when the punctuation is at 60 characters. It's also difficult to tell if the original author intended it to be two separate paragraphs and made a mistake or it should be one.